Gems/AzQtComponentsForPython/bindings_azqtcomponents.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+ 
+#pragma once
+/*
+#include <QPointer>
+
+//#include <AzCore/RTTI/TypeInfo.h>
+//#include <AzCore/Math/MathUtils.h>
+#include <AzCore/Math/Color.h>
+#include <AzCore/std/function/function_template.h>
+#include <AzCore/std/chrono/chrono.h>
+*/
+
+//#include <AzQtComponents/Components/AutoCustomWindowDecorations.h>
+
+// Ensure the correct AZ_VE_NUM_ARGS macro is used inside variadic.h
+#define FORCE_VA_OPT_ARG 1
```

Comment:
Why is this required exactly?

Reply:
It was to work around a bug caused by shiboken automatically adding an option to clang which caused the compile to fail. There's a patched version of the exe in progress that fixes this so I've removed this line.